samples/table_fetch_rows_with_limit.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+
+def fetch_rows_with_limit(table):
+    """Fetch rows from the table with limit"""
+
+    # TODO(developer): Create the table
+    # table = Table(
+    #    table_name,
+    #    metadata,
+    #    Column("user_id", Integer, primary_key=True),
+    #    Column("user_name", String(16), nullable=False),
+    # )
+    # table.create()
+
+    table.insert().execute([
```

Comment:
This should not be here; this belongs in the test.

The sample functions should be contained within the sample tags, i.e.
```
# [START sample_tag]
def function(arg):
    ...
# [END sample_tag]
```

This means there shouldn't be any return values.